include/pprof/Tracing.h

```diff
@@ -1,6 +1,7 @@
 #ifndef PPROF_TRACING_H
 #define PPROF_TRACING_H
 
+#include "string.h"
```

Comment:
I'm not 100% sure that this also solves the error, but I think you want to include <cstring> instead of string.h.